src/components/FormModal.jsx

```diff
@@ -0,0 +1,52 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faXmark } from '@fortawesome/free-solid-svg-icons';
+
+export default function FormModal({
+	handleSubmit,
+	closeForm,
+	existingToken,
+	handleChange,
+	errorMessage,
+}) {
+	return (
+		<div className="bg-[rgba(0,0,0,0.4)] w-screen h-screen flex justify-center items-center fixed">
+			<form
+				onSubmit={handleSubmit}
+				className="bg-white h-64 flex flex-col items-start justify-center gap-4 rounded-md px-10"
+			>
+				<FontAwesomeIcon
+					icon={faXmark}
+					className="text-xl self-end cursor-pointer hover:text-main"
+					onClick={closeForm}
+				></FontAwesomeIcon>
+				<label htmlFor="existingToken" className="text-xl font-bold">
+					Want to join an existing list?
+					<p className="text-base font-normal mb-6">
+						Enter the list's three word token below
+					</p>
+					<input
+						type="text"
+						id="existingToken"
+						value={existingToken}
+						onChange={handleChange}
+						required
+						className="text-base font-normal w-[90%] border-main border-2 mb-4"
```

Comment:
```suggestion
						className="text-base font-normal w-[90%] border-main border-2 mb-4 p-2"
```

I suggest giving the text input some padding so that there is some space between the text inside and the box.